cockroachdb/manifests-cert-manager/service.yaml

```diff
@@ -37,7 +37,7 @@ metadata:
     # Enable automatic monitoring of all instances when Prometheus is running in the cluster.
     prometheus.io/scrape: "true"
     prometheus.io/path: "_status/vars"
-    prometheus.io/port: "8001"
+    prometheus.io/port: "8080"
```

Comment:
for clarity: is this just for consistency? I.e. use the same port as everyone else to keep things simpler?

Reply:
CRDB has a metrics endpoint exposed in `:8080/_status/vars`.
I believe previously we used another container (`certs-refresh`) to scrap metrics, but as that one is gone since migrating to cert-manager we need to point prometheus to the main container for metrics.